pom.xml

```diff
@@ -31,8 +31,15 @@
         <munit.input.directory>src/test/munit</munit.input.directory>
         <munit.citizen.input.directory>src/test/munit-citizen</munit.citizen.input.directory>
         <munit.output.directory>${basedir}/target/test-mule/munit</munit.output.directory>
-        <munit.extensions.maven.plugin.version>1.2.0-SNAPSHOT</munit.extensions.maven.plugin.version>
-        <munit.version>3.1.0-SNAPSHOT</munit.version>
+        <munit.extensions.maven.plugin.version>1.2.0-rc1</munit.extensions.maven.plugin.version>
+        <munit.version>3.1.0-rc2</munit.version>
+        <mtf.tools.version>1.2.0-rc1</mtf.tools.version>
+
+        <mule.http.connector.version>1.9.0-SNAPSHOT</mule.http.connector.version>
```

Comment:
If this is going to be a GA version(and in my opinion it should be), make sure that it does not use SNAPSHOT versions and that all changes will be merged to support/2.0.x and released in 2.0.1. Regards!